README.md

```diff
@@ -13,122 +13,293 @@ a demo [online](https://www.e-spirit.com/us/specialpages/forms/on-demand-demo/).
 
 ## Getting Started
 
-### Create `.env` file in root-directory
-
-```bash
-# APIKey used for authentication against the CaaS
-FSXA_API_KEY= ...
-# URL pointing to your CaaS Instance
-FSXA_CAAS= ...
-# ID of your project
-FSXA_PROJECT_ID= ...
-# URL pointing to the NavigationService
-FSXA_NAVIGATION_SERVICE= ...
-# Which mode should be used? (preview/release)
-FSXA_MODE= ...
-# KEY:VALUE map where semicolon is used as separator (key:uuid;key:uuid)
-FSXA_REMOTES= ...
-# This tenantId is required, when using caas-connect module 3.0.9 and above
-FSXA_TENANT_ID=...
+This chapter describes how to set up the project and complete the first steps.
+
+### Requirements
+
+- [node](https://nodejs.org/en/) - latest LTS Version: 14.15.5.
```

Comment:
Should we really write down such a specific version? Today node.js offers _14.16.0 LTS_ or _15.11.0 Current_. Couldn't we write 14.x.x LTS? Or maybe:
"latest LTS version (> 14.15.0)"